### ISSUE TITLE
Cannot import package due to exported js files not being modules

### ISSUE THREAD

**Issue by josh-signal** [KittyGiraudel/focusable-selectors#15]
package.json specifies `type: "module"` but the distributed js files are using `require` so node is throwing when trying to import the package.

See [Node docs ESM](https://nodejs.org/docs/latest-v13.x/api/esm.html#esm_enabling)

**Comment by KittyGiraudel** [comment]
Hello. The package exports both a CJS and an ESM entry point, and I use it in another library of mine, so I’m not sure there is an issue with the package itself. Maybe if you share some code, I can check. :)

**Comment by josh-signal** [comment]
```js
const focusableSelectors = require('focusable-selectors');

console.log(focusableSelectors);
```

On Node v18.14.0 try running the file.

```bash
node focusableSelectorsTest.js
```

```
./focusableSelectorsTest.js:1
const focusableSelectors = require('focusable-selectors');
                           ^

Error [ERR_REQUIRE_ESM]: require() of ES Module ./node_modules/focusable-selectors/index.js from ./focusableSelectorsTest.js not supported.
index.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
Instead rename index.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in ./node_modules/focusable-selectors/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).

    at Object.<anonymous> (./focusableSelectorsTest.js:1:28) {
  code: 'ERR_REQUIRE_ESM'
}
```

**Comment by KittyGiraudel** [comment]
I can reproduce. I think we need to remove `"type": "module"` from the package.

**Comment by KittyGiraudel** [comment]
Fixed in 0.8.4. Sorry for the delay. :(
